packages/docs/site/i18n/ja/docusaurus-plugin-content-docs/current/blueprints/tutorial/01-what-are-blueprints-what-you-can-do-with-them.md

```diff
@@ -0,0 +1,150 @@
+---
+title: ブループリントとは何ですか?
+slug: /blueprints/tutorial/what-are-blueprints-what-you-can-do-with-them
+description: ブループリントを使い始める
```

Comment:
Learn what Blueprints are and how they configure WordPress Playground. Discover the benefits of using JSON for instant site setup.

Reply:
Thanks, I fixed the original in another branch.
https://github.com/WordPress/wordpress-playground/pull/2512

Reply:
Thank you, I updated the description.